docs/content/latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-apps-java/cloud-ysql-yb-jdbc.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Build a Java application that uses YSQL
+headerTitle: Build a Java application
+linkTitle: Java
+description: Build a simple Java application using the YugabyteDB JDBC Driver and using the YSQL API to connect to and interact with a Yugabyte Cloud cluster.
+menu:
+  latest:
+    parent: cloud-build-apps
+    name: Java
+    identifier: cloud-java-1
+    weight: 550
+type: page
+isTocNested: true
+showAsideToc: true
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+  <li >
+    <a href="../ysql-yb-jdbc/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - YB - JDBC
+    </a>
+  </li>
+</ul>
+
+The application connects to your Yugabyte Cloud instance through the topology-aware [Yugabyte JDBC driver](../../../../../integrations/jdbc-driver/) and performs basic SQL operations. Use the application as a template to get started with Yugabyte Cloud in Java.
+
+## Prerequisites
+
+This tutorial assumes the following:
+
+- You have a cluster deployed in Yugabte Cloud. To get started, use the [Quick start](../../../).
+- You downloaded the cluster CA certificate. Refer to [Download your cluster certificate](../../../../cloud-secure-clusters/cloud-authentication/#download-your-cluster-certificate).
+- You have added your computer to the cluster IP allow list. Refer to [Assign IP Allow Lists](../../../../cloud-secure-clusters/add-connections/).
+- Java Development Kit (JDK) 1.8, or later, is installed. JDK installers for Linux and macOS can be downloaded from [Oracle](http://jdk.java.net/), [Adoptium (OpenJDK)](https://adoptium.net/), or [Azul Systems (OpenJDK)](https://www.azul.com/downloads/?package=jdk). Homebrew users on macOS can install using `brew install openjdk`.
+- [Apache Maven](https://maven.apache.org/index.html) 3.3 or later, is installed.
+
+## Clone the application from GitHub
+
+Clone the sample application to your computer:
+
+```sh
+git clone https://github.com/yugabyte/simple-java-app-yugabyte-cloud.git && cd simple-java-app-yugabyte-cloud
```

Comment:
Made this application generic so that we can use it for both Cloud and Core quick start guides. As a result, changed the repo name and updated slightly the GitHub README file. Check it out and update on your side:
https://github.com/yugabyte/yugabyte-simple-java-app